src/Cardano/Shell/Constants/CLI.hs

```diff
@@ -19,13 +19,16 @@ import           Cardano.Shell.Constants.Types
 lastOption :: Parser a -> Parser (Last a)
 lastOption parser = Last <$> optional parser
 
+lastStrOption :: IsString a => Mod OptionFields a -> Parser (Last a)
```

Comment:
My eyes :eye: 
Do we really need such micro improvements?
They sometimes introduce more complexity at the price of a little duplication.

Some duplication is generally ok.

Reply:
The idea is that _all_ the `strOption`'s will end up needing a `Last` wrapper -- that's in the nature of the approach.  Unless I'm missing something, of course..